worker/src/main/scala/org/apache/celeborn/service/deploy/worker/storage/StorageManager.scala

```diff
@@ -456,7 +496,9 @@ final private[worker] class StorageManager(conf: CelebornConf, workerSource: Abs
     isHdfsExpired
   }
 
-  def cleanupExpiredShuffleKey(expiredShuffleKeys: util.HashSet[String]): Unit = {
+  def cleanupExpiredShuffleKey(
+      expiredShuffleKeys: util.HashSet[String],
+      onClose: Boolean = false): Unit = {
```

Comment:
IMO we should change `onClose` to `cleanDb`, and only set it to true in `Worker.cleanup`